Lib/Neon.Kube.Setup/KubeSetup.Operations.cs

```diff
@@ -2639,7 +2639,7 @@ private static async Task DeployOpenEbsWithcStor(ISetupController controller, No
                             },
                             PoolConfig = new V1CStorPoolConfig()
                             {
-                                DataRaidGroupType = DataRaidGroupType.Stripe,
```

Comment:
Jeff and I talked about this recently. The issue here is the k8s client not respecting the `EnumMember` properties. We decided that the best solution is to just use constant strings instead here.